G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs

```diff
@@ -112,7 +112,10 @@ private IWebElement FindElement(string search, string by, TimeSpan timeout)
                         element = new WebDriverWait(webDriver, timeout).Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementExists(By.Id(search)));
                         break;
                     case ElementSearchBy.Class:
-                        element = new WebDriverWait(webDriver, timeout).Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementExists(By.ClassName(search)));
+                        if (search.Contains(" "))
+                            element = new WebDriverWait(webDriver, timeout).Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementExists(By.XPath($"//*[@class='{search}']")));
+                        else
+                            element = new WebDriverWait(webDriver, timeout).Until(SeleniumExtras.WaitHelpers.ExpectedConditions.ElementExists(By.ClassName(search)));

```

Comment:
OMG, get rid of those long namespace prefixes of classes.
Also, consider this change, it will help to avoid some DRY breakage:
```csharp
                        element = new WebDriverWait(webDriver, timeout).Until(
                            ExpectedConditions.ElementExists(
                                search.Contains(" ") ? By.XPath($"//*[@class='{search}']") : By.ClassName(search)
                            )
                        );
```